iothub/device/src/InternalClient.cs

```diff
@@ -1885,6 +1885,77 @@ internal void ValidateModuleTransportHandler(string apiName)
 
         #endregion Module Specific API
 
+        #region Convention driven operations
+
+        internal async Task<Properties> GetPropertiesAsync(CancellationToken cancellationToken)
+        {
+            try
+            {
+                Twin hubTwin = await InnerHandler.SendTwinGetAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
this is simply converting a twin to properties for now, but this will be reimplemented (since response -> twin -> property brings in deserialization with newtonsoft into picture)